src/test/enhetstester/kotlin/no/nav/familie/ba/sak/config/AbstractMockkSpringRunner.kt

```diff
@@ -92,12 +92,12 @@ abstract class AbstractMockkSpringRunner {
     }
 
     fun settToggleMock(
-        toggle: String,
+        toggle: FeatureToggle,
         value: Boolean,
     ) {
-        every { mockUnleashService.isEnabled(toggle) } returns value
-        every { mockUnleashService.isEnabled(toggle, defaultValue = any()) } returns value
-        every { mockUnleashService.isEnabled(toggle, properties = any()) } returns value
+        every { mockFeatureToggleService.isEnabled(toggle) } returns value
+        every { mockFeatureToggleService.isEnabled(toggle = toggle, behandlingId = any<Long>()) } returns value
```

Comment:
Ser `properties` er endret her. Er det meningen? 

```suggestion
        every { mockFeatureToggleService.isEnabled(toggle = toggle, properties = any<Long>()) } returns value
```